grouper/fe/template_util.py

```diff
@@ -41,14 +46,17 @@ def print_date(date_obj):
 
     Returns human readable date/time string.
     """
-    if date_obj is None:
+    if date_obj is None or date_obj == '':
         return ""
 
     date_obj = _make_date_obj(date_obj)
 
-    date_obj = date_obj.astimezone(settings["timezone"])
-    return date_obj.strftime(settings["date_format"])
-
+    if date_obj:
+        assert isinstance(date_obj, datetime)
+        date_obj = date_obj.astimezone(settings["timezone"])
+        return date_obj.strftime(settings["date_format"])
+    else:
+        return ""
```

Comment:
This check would be unnecessary if `_make_date_obj` always returns a `datetime`.